src/net_processing.cpp

```diff
@@ -3164,7 +3167,14 @@ bool SendMessages(CNode* pto, CConnman& connman, const std::atomic<bool>& interr
                 return true;
             }
         }
+        MsgGetData(consensusParams, pto, connman, msgMaker, state, nNow, fFetch);
+        MsgFeeFilter(pto, connman, msgMaker);
+    }
+    return true;
+}
 
+void MsgGetData(const Consensus::Params& consensusParams, CNode* pto, CConnman& connman, const CNetMsgMaker& msgMaker, CNodeState& state, int64_t nNow, bool fFetch)
```

Comment:
Long time ago, I remember I was asked not to do this for basic types. Recently we discussed a PR to change this in the whole codebase (sorry, I can't find it). But good good point, no reason not to do it here and in #9608. It also helps me read functions when more parameters are const.